views/admin/must-connect-main-blog.php

```diff
@@ -4,7 +4,7 @@
 			<div class="jetpack-text-container">
 				<h2><?php _e( 'Get started with Jetpack Multisite', 'jetpack' ); ?></h2>
 				<h4>
```

Comment:
In a perfect world, it would be better if the headings did not skip levels, from two to four. That way, the heading structure is more hierarchical and the `h4` wouldn't be used strictly for appearance sake.